### PR TITLE
updates fo cookieclicker v2.048 #1049

### LINKED ISSUE #1078: Bonus income calculation is incorrect
### Bug description

When game is loaded without any 2.048 unlocks unlocked (no cortex bakers, no unshackled unlocks, no new upgrades) bonus income calculations are not accurate to actual income change after purchase.

in screenshot below, calculations for change after purchasing one grandma is off by 6.6%

income before purchase: 6.630452138918733e+50
income after purchase: 6.634013662707543e+50

actual bonus income: 3.56152378881e+47
calculated bonus income: 3.333663966254572e+47

### To reproduce

1. load game
2. open console and run Game.cookiesPs to get current cps
3. check CookieMonsterData for bonus income on a building
4. purchase  building
5. open console and run Game.cookiesPs to get new cps
6. compare

note: can use Game.cookiesPsRaw to avoid any active golden cookie buffs

### Screenshot

image

### Save file

Mi4wNDN8fDE2NTA1MDU5MzAyMDc7MTYwODE3NTc5MzMwMTsxNjU0MDQ1OTA5OTkxO0hlbGxhIENsaWNrO2JjdndofDAwMTEwMDAwMTAwMDAwMDAwMTAwMDExMDAwfDIuODI1NjU3ODI0OTM5MTgzN2UrNjI7MS4zNzIwNzQxNTg4NjczNTU5ZSs2Mzs2NTg4MTM7ODUyNzM7MS4zNjk3ODYzMDU0NTc4NTQ5ZSs2MzsxMjk1NjsxOzA7NC43MjUwODI3ODM1NjkzNTJlKzYyOzM7MTswOzA7LTE7MTAwNTszMDg1OzQuMzMzOTQ2MTAxMDk1NDE1ZSs1NjsxMDc7MTQ7MzM3Oy0xOzg7OzUuMTk3MjM2MzE5MzQ5ODQ4ZSs1NzsxMjs3Nzg4Nzg2NjM5NjIxNjM3MDs3Nzg4NTYyOTE0ODQ0MDI5MDsyMjM3MjQ3Nzc2MDkyOzA7MDsyMjY7NzE2OzY0MTs2OTk7NjEzOzI1OzE4OzE3OzA7MTQ7MDswOzE1Nzs4NDI7MTY1NDAwMjMzNzkwMTswOzA7OzgzOzE7MTs2LjA5Njc3NjUxNzkxNDg1N2UrNTA7MDt8OTg1LDEzMzUsMS45Njg5NTI2NDI5MzUxMTFlKzU3LDEyLCwwLDk4NTs5ODYsMTMzNiwyLjA5OTk3ODgyNTAxOTAzNThlKzU3LDEwLCwxLDk4Njs5MzYsMTI4Niw3LjMxODE1MjUwMTQ4Mzg1NmUrNTIsMTAsMTY1NDA0NjEwMzcwNTo0OjE2NTQwNDMzMTIwNTQ6MDoxMTI0OjUzOTg6MToxMDoxNjUwNTA1OTMwMjkxOiAxMTExMTExMTExMTExMTExMTExMTEwMTExMTExMTExMTAxIDg6MTo4OjI6ODoxOjg6Mjo4OjI6ODoyOjg6MTo4OjI6ODoyOjg6MDo4OjA6ODoyOjg6MDo4OjE6ODowOjg6MTo4OjE6ODoyOjg6MTo4OjE6ODo4Njo4Ojg2Ojg6ODY6ODo4Njo4OjI6MzM6Mjo4Ojg2Ojg6ODY6MjI6NDQ6ODo4Njo4OjE6ODoxOjg6ODY6ODo4Njo4Ojg2Ojg6ODY6LDAsOTM2OzkyMiwxMjcyLDcuOTk3ODU5NjMwMzUxMTllKzU0LDEsLDEsOTIyOzkwMiwxMjUyLDEuMjY1NjYwNjI3MTk5MDU0ZSs1NSwxLCwxLDkwMjs4ODUsMTIzNSwyLjQ5NDk2MjIzNDE5NDY1MTZlKzU0LDcsMDowOjE6MDowOiA0MjQ6NDotNjI6MTcwOjA6MDowITMyNzo1Oi0xMTI6NDUyOjA6MDowITI3NTI6NTo3NzozMjI6MDowOjAhNzUwODoxOjU2Ojc4OjA6MDowITY1Mzo0Oi0xNDU6Mzk3OjA6MDowITczNjU6NTo0MzoxNDk6MDowOjAhNzgyMDowOi0zOjE4MTowOjA6MCE4NTk2OjU6Mzo5MDowOjA6MCEzMDI3OjQ6LTIwMzoxMTc6MDowOjAhMTA3MzU6NTo3MTo0NTowOjA6MCEyODQ1OjQ6LTIwMTo3NDQ6MDowOjAhMTI3ODM6MDo2OjE3MDowOjA6MCExNDE2MToxOjg6MTc0OjA6MDowITEyMDgwOjI6LTU5OjQ5MzowOjA6MCExMzIzMDoyOi01NTo1MzU6MDowOjAhMTM4NjQ6MjotNTA6NTkyOjA6MDowISAwLDEsODg1Ozg2NiwxMjE2LDEuMDc5NTczMTMwMzE5Mjg3M2UrNTUsMywxMC84LzAgMyAxNjUxMzY2NTgwNjM4IDAsMCw4NjY7ODQzLDgwOTMsMS4wMTI4NjczMDQ0MTQwMTUxZSs1MywxLDEyMSA0NTMgNTM5NiAwLDEsODQzOzgyNiwxMTc2LDYuMzM3MTcwMTAyMjA3MzdlKzUzLDAsLDEsODI2OzgwNywxMTU3LDEuODk2MzI2NjA3NTA5NTM2ZSs1NCw0LCwwLDgwNzs4MDgsMTE1OCw2LjU3OTk0NDk5Njg5OTk3NWUrNTYsMTAsLDEsODA4Ozc3NSwxMTI1LDEuMjQxNjc4Nzk2OTg3Mzg4N2UrNTYsMTAsLDEsNzc1Ozc2MCwxMTEwLDEuMzg2NDkyNDQ0NzI3MjE0ZSs1NywxMCwsMSw3NjA7NzUyLDExMDIsNC41ODEwOTA3MDgyMDQwOTJlKzU3LDEwLCwxLDc1Mjs3MzEsMTA4MSwxLjY0NjYyMTAxMDAxOTg4ODdlKzU4LDEwLCwxLDczMTs3MjcsMTA3NywxLjEzMjA0NDYyOTcyNzc4OTRlKzU5LDEwLCwxLDcyNzs2NzgsMTAyOCwyLjI1ODI5OTAwNTU3MzcwNTFlKzU4LDEwLCwxLDY3ODs2NTIsMTAwNiwxLjQzOTM0MDYxMjcyNzM3MDFlKzU5LDEwLCwxLDY1NDt8MTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTEwMTExMTExMTEwMDExMTExMTAwMTAxMTExMDAxMTExMTEwMDExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTEwMDExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTEwMDExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTAwMTExMTExMTExMTExMTExMTExMTExMTExMTExMTEwMTAxMDEwMDAxMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTEwMDEwMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTEwMTAxMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMDExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTAwMDAxMTExMTExMTExMTExMTExMTExMTExMTExMTExMTAxMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTEwMTExMTExMTExMTExMDAxMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTAxMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTAwMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExfDExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTEwMTExMTExMTExMTExMTExMTExMTExMTExMTAwMDAxMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTAxMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTEwMDAwMDAwMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTExMTF8fENvb2tpZU1vbnN0ZXI6eyJzZXR0aW5ncyI6eyJDUFNNb2RlIjowLCJBdmdDUFNIaXN0IjowLCJBdmdDbGlja3NIaXN0IjowLCJDYWxjV3JpbmsiOjEsIlNjYWxlIjoyLCJTY2FsZURlY2ltYWxzIjoyLCJTY2FsZVNlcGFyYXRvciI6MCwiU2NhbGVDdXRvZmYiOjk5OTk5OSwiVGltZUZvcm1hdCI6MCwiRGV0YWlsZWRUaW1lIjoxLCJQUERpc3BsYXlUaW1lIjowLCJCdWlsZENvbG91ciI6MSwiUFBPbmx5Q29uc2lkZXJCdXlhYmxlIjowLCJQUEV4Y2x1ZGVUb3AiOjAsIlBQUmlnaWRlbE1vZGUiOjEsIlBQU2Vjb25kc0xvd2VyTGltaXQiOjAsIkNvbG91ckJsdWUiOiIjNGJiOGYwIiwiQ29sb3VyR3JlZW4iOiIjMDBmZjAwIiwiQ29sb3VyWWVsbG93IjoiI2ZmZmYwMCIsIkNvbG91ck9yYW5nZSI6IiNmZjdmMDAiLCJDb2xvdXJSZWQiOiIjZmYwMDAwIiwiQ29sb3VyUHVycGxlIjoiI2ZmMDBmZiIsIkNvbG91ckdyYXkiOiIjYjNiM2IzIiwiQ29sb3VyUGluayI6IiNmZjE0OTMiLCJDb2xvdXJCcm93biI6IiM4YjQ1MTMiLCJCb3RCYXIiOjAsIlRpbWVyQmFyIjoxLCJUaW1lckJhclBvcyI6MSwiVGltZXJCYXJPdmVybGF5IjoxLCJBdXRvc2F2ZVRpbWVyQmFyIjowLCJVcEJhckNvbG91ciI6MSwiVXBncmFkZUJhckZpeGVkUG9zIjoxLCJTb3J0QnVpbGRpbmdzIjoyLCJTb3J0VXBncmFkZXMiOjEsIlVwZ3JhZGVzTmV2ZXJDb2xsYXBzZSI6MCwiRHJhZ29uQXVyYUluZm8iOjEsIkdyaW1vaXJlQmFyIjoxLCJHQ1RpbWVyIjoxLCJGYXZpY29uIjoxLCJXcmlua2xlckJ1dHRvbnMiOjEsIkhpZGVTZWN0aW9uc0J1dHRvbnMiOjAsIlRvb2x0aXBCdWlsZFVwZ3JhZGUiOjEsIlRvb2x0aXBBbW9yIjowLCJUb29sV2Fybkx1Y2t5IjowLCJUb29sV2Fybkx1Y2t5RnJlbnp5IjowLCJUb29sV2FybkNvbmp1cmUiOjEsIlRvb2xXYXJuQ29uanVyZUZyZW56eSI6MSwiVG9vbFdhcm5FZGlmaWNlIjowLCJUb29sV2FyblVzZXIiOjAsIlRvb2xXYXJuQm9uIjoxLCJUb29sV2FyblBvcyI6MSwiVG9vbHRpcEdyaW0iOjEsIlRvb2x0aXBXcmluayI6MCwiVG9vbHRpcEx1bXAiOjEsIlRvb2x0aXBQbG90cyI6MSwiVG9vbHRpcFBhbnRoZW9uIjoxLCJUb29sdGlwQXNjZW5kQnV0dG9uIjoxLCJTdGF0cyI6MSwiTWlzc2luZ1VwZ3JhZGVzIjoxLCJNaXNzaW5nQWNoaWV2ZW1lbnRzIjowLCJVcFN0YXRzIjowLCJIZWF2ZW5seUNoaXBzVGFyZ2V0IjoxLCJTaG93TWlzc2VkR0MiOjEsIlRpdGxlIjoxLCJHZW5lcmFsU291bmQiOjEsIkdDTm90aWZpY2F0aW9uIjowLCJHQ0ZsYXNoIjoxLCJDb2xvdXJHQ0ZsYXNoIjoiI2ZmZmZmZiIsIkdDU291bmQiOjEsIkdDVm9sdW1lIjoxMDAsIkdDU291bmRVUkwiOiJodHRwczovL2ZyZWVzb3VuZC5vcmcvZGF0YS9wcmV2aWV3cy82Ni82NjcxN185MzE2NTUtbHEubXAzIiwiRm9ydHVuZU5vdGlmaWNhdGlvbiI6MCwiRm9ydHVuZUZsYXNoIjoxLCJDb2xvdXJGb3J0dW5lRmxhc2giOiIjZmZmZmZmIiwiRm9ydHVuZVNvdW5kIjoxLCJGb3J0dW5lVm9sdW1lIjoxMDAsIkZvcnR1bmVTb3VuZFVSTCI6Imh0dHBzOi8vZnJlZXNvdW5kLm9yZy9kYXRhL3ByZXZpZXdzLzE3NC8xNzQwMjdfMzI0MjQ5NC1scS5tcDMiLCJTZWFOb3RpZmljYXRpb24iOjAsIlNlYUZsYXNoIjoxLCJDb2xvdXJTZWFGbGFzaCI6IiNmZmZmZmYiLCJTZWFTb3VuZCI6MSwiU2VhVm9sdW1lIjoxMDAsIlNlYVNvdW5kVVJMIjoiaHR0cHM6Ly93d3cuZnJlZXNvdW5kLm9yZy9kYXRhL3ByZXZpZXdzLzEyMS8xMjEwOTlfMjE5MzI2Ni1scS5tcDMiLCJHYXJkRmxhc2giOjEsIkNvbG91ckdhcmRGbGFzaCI6IiNmZmZmZmYiLCJHYXJkU291bmQiOjEsIkdhcmRWb2x1bWUiOjEwMCwiR2FyZFNvdW5kVVJMIjoiaHR0cHM6Ly9mcmVlc291bmQub3JnL2RhdGEvcHJldmlld3MvMTAzLzEwMzA0Nl84NjE3MTQtbHEubXAzIiwiTWFnaWNOb3RpZmljYXRpb24iOjAsIk1hZ2ljRmxhc2giOjAsIkNvbG91ck1hZ2ljRmxhc2giOiIjZmZmZmZmIiwiTWFnaWNTb3VuZCI6MCwiTWFnaWNWb2x1bWUiOjEwMCwiTWFnaWNTb3VuZFVSTCI6Imh0dHBzOi8vZnJlZXNvdW5kLm9yZy9kYXRhL3ByZXZpZXdzLzIyMS8yMjE2ODNfMTAxNTI0MC1scS5tcDMiLCJXcmlua2xlck5vdGlmaWNhdGlvbiI6MCwiV3JpbmtsZXJGbGFzaCI6MCwiQ29sb3VyV3JpbmtsZXJGbGFzaCI6IiNmZmZmZmYiLCJXcmlua2xlclNvdW5kIjowLCJXcmlua2xlclZvbHVtZSI6MTAwLCJXcmlua2xlclNvdW5kVVJMIjoiaHR0cHM6Ly9mcmVlc291bmQub3JnL2RhdGEvcHJldmlld3MvMTI0LzEyNDE4Nl84MDQzLWxxLm1wMyIsIldyaW5rbGVyTWF4Tm90aWZpY2F0aW9uIjowLCJXcmlua2xlck1heEZsYXNoIjowLCJDb2xvdXJXcmlua2xlck1heEZsYXNoIjoiI2ZmZmZmZiIsIldyaW5rbGVyTWF4U291bmQiOjAsIldyaW5rbGVyTWF4Vm9sdW1lIjoxMDAsIldyaW5rbGVyTWF4U291bmRVUkwiOiJodHRwczovL2ZyZWVzb3VuZC5vcmcvZGF0YS9wcmV2aWV3cy8xNTIvMTUyNzQzXzE1NjYzLWxxLm1wMyIsIkJ1bGtCdXlCbG9jayI6MSwiRmF2b3VyaXRlU2V0dGluZ3MiOjJ9LCJoZWFkZXJzIjp7IkZhdm91cml0ZSI6MSwiQ2FsY3VsYXRpb24iOjAsIk5vdGF0aW9uIjowLCJDb2xvdXJzIjowLCJCYXJzRGlzcGxheSI6MCwiVG9vbHRpcCI6MCwiU3RhdGlzdGljcyI6MCwiTm90aWZpY2F0aW9uIjowLCJOb3RpZmljYXRpb25HZW5lcmFsIjowLCJOb3RpZmljYXRpb25HQyI6MCwiTm90aWZpY2F0aW9uRkMiOjAsIk5vdGlmaWNhdGlvblNlYSI6MCwiTm90aWZpY2F0aW9uR2FyZCI6MCwiTm90aWZpY2F0aW9uTWFnaSI6MCwiTm90aWZpY2F0aW9uV3JpbmsiOjAsIk5vdGlmaWNhdGlvbldyaW5rTWF4IjowLCJNaXNjZWxsYW5lb3VzIjowLCJMdWNreSI6MCwiQ2hhaW4iOjEsIlNwZWxscyI6MCwiR2FyZGVuIjowLCJQcmVzdGlnZSI6MCwiV3JpbmsiOjAsIlNlYSI6MSwiQWNoaWV2cyI6MCwiTWlzYyI6MCwiaW5mb01lbnUiOjEsIm9wdGlvbnNNZW51IjowfSwiZmF2b3VyaXRlU2V0dGluZ3MiOltdLCJsb2NrZWRNaW5pZ2FtZXMiOltdLCJ2ZXJzaW9uIjoiMi4wMzEuMTAifTtjb29raWVNb25zdGVyRnJhbWV3b3JrOnsic2V0dGluZ3MiOnt9LCJoZWFkZXJzIjp7ImluZm9NZW51IjoxLCJvcHRpb25zTWVudSI6MX0sImZhdm91cml0ZVNldHRpbmdzIjpbXX07Q0NTRTp7InZlcnNpb24iOiIyLjAzMSIsIkFjaGlldmVtZW50cyI6e30sIlVwZ3JhZGVzIjp7fSwiQnVpbGRpbmdzIjp7fSwiQnVmZnMiOnt9LCJTZWFzb25zIjp7fSwiT3RoZXJNb2RzIjp7fSwidmF1bHQiOltdLCJwZXJtYW5lbnRVcGdyYWRlcyI6Wy0xLC0xLC0xLC0xLC0xXSwiY2hpbWVUeXBlIjoiQ2hpbWUiLCJtaWxrVHlwZSI6IkF1dG9tYXRpYyIsImJnVHlwZSI6IkJsdWUifTtDaG9vc2UgeW91ciBvd24gbHVtcDp7InZlcnNpb24iOiIxLjMuMSIsInNldHRpbmdzIjp7ImRpc2NyZXBhbmN5IjoxLCJpbmNsdWRlTm9ybWFsIjpmYWxzZSwiaW5jbHVkZUJpZnVyY2F0ZWQiOmZhbHNlLCJpbmNsdWRlR29sZGVuIjp0cnVlLCJpbmNsdWRlTWVhdHkiOmZhbHNlLCJpbmNsdWRlQ2FyYW1lbGl6ZWQiOmZhbHNlLCJwcmVzZXJ2ZUdyYW5kbWFwb2NhbHlwc2VTdGFnZSI6ZmFsc2UsInByZXNlcnZlRHJhZ29uIjp0cnVlLCJwcmVzZXJ2ZVBhbnRoZW9uIjpmYWxzZSwicm93c1RvRGlzcGxheSI6MTB9fTtIb3J0aWNvb2tpZTp7ImFoTmV3IjoxLCJhaEp1aWN5IjoxLCJhaFVwZ3JhZGUiOjB9O01FVEE6Y29va2llIG1vbnN0ZXIsKkNDU0UsKkhvcnRpY29va2llLCpDaG9vc2UgeW91ciBvd24gbHVtcCwqY29vbGVyIHNhbXBsZSBtb2QsKmxhbmcgc2FtcGxlIG1vZCwqc2FtcGxlIG1vZDs%3D%21END%21

### Browser / Steam / Version

browser v2.043 and v2.048

### PR DESCRIPTION
**Pull request by domoddball**
Resolves negative calculation issues introduced by most recent cookie clicker update (v2.048) 
additionally a few updates for achievement requirements that have been changed 

does not support unshackled upgardes
greater calculation bug seems to exist with calculations, is exacerbated with unshackled upgrades 

Closes #1078

### CODE REVIEW

**Comment by DanielNoord** [comment]
Could you also rebuild the output files?

**Comment by domoddball** [comment]
My apologies, i thought i had properly rebuilt everything
which output files do i need to rebuild?

**Comment by DanielNoord** [comment]
Oh never mind. The rebuild is correct but `package-lock` isn't. Sorry I reviewed from mobile.

Could you reset the `package-lock`?

**Comment by domoddball** [comment]
again my apologies, but im not particularly familiar with js and its terminology 
by "reset the `package-lock` " do you mean to revert the changes that were introduced by the `npm install` command?
or is there another set of commands that should be run?

**Comment by DanielNoord** [comment]
> again my apologies, but im not particularly familiar with js and its terminology
> by "reset the `package-lock` " do you mean to revert the changes that were introduced by the `npm install` command?
> or is there another set of commands that should be run?

I must confess I'm also mostly working in other environments.
What seems to happen is you are using and old version of `package-lock.json`. See `"lockfileVersion": 1,` whereas this project (and most others) seem to be using `2`. I'm not sure how this can come about, perhaps you version of `npm` is outdated? 
The changes in this PR shouldn't amount to a diff of over 4000 lines changes. Which all come from changes to the `packages-lock.json` file.

**Comment by domoddball** [comment]
` perhaps you version of npm is outdated`
this hit the nail on the head. updated npm version and issue resolved

File: src/Data/Gamedata.ts [hunk 1]
```diff
@@ -20,6 +20,7 @@ export const Fortunes: string[] = [
   'Fortune #016',
   'Fortune #017',
   'Fortune #018',
+  'Fortune #019',
```
Comment (DanielNoord) [inline src/Data/Gamedata.ts:23]: I don't see a new Fortune `#019` in http://orteil.dashnet.org/cookieclicker/main.js?v=2.099
Am I missing something?

File: src/Data/Moddata.ts [hunk 2]
```diff
@@ -12,14 +12,7 @@ export const ModDescription = `<a href="https://github.com/CookieMonsterTeam/Coo
...
-- Added a percentage to the Golden Cookie timer bar</br>
-</br>
-This update fixes the following bugs:
-- Fix considerable lag on the Ascension screen when using the monospace font</br>
-- Fix sound playing at start-up</br>
-- Fix building tooltips and warnings not updating correctly</br>
-- Fix upgrade bar not displaying</br>
-- Fix loading of mod when no save was found</br>`;
+export const LatestReleaseNotes = `This update adds support for some parts of cookie clicker 2.048</br>
+- added support for Dragin Aura 'Supreme Intellect' in the pantheon calculations</br>
```
Comment (DanielNoord) [inline src/Data/Moddata.ts]:
```suggestion
- added support for Dragon Aura 'Supreme Intellect' in the pantheon calculations</br>
```

**Review by DanielNoord — COMMENTED** [review]
One question, rest looks good to me!

File: src/Data/Gamedata.ts (hunk 1 above)
Comment (domoddball) [reply src/Data/Gamedata.ts:23]: line 11259

image

Comment (DanielNoord) [reply src/Data/Gamedata.ts:23]: Oh weird... My `ctrl + f` is not good enough it seems 😄

**Review by DanielNoord — CHANGES_REQUESTED** [review]
You probably need to rebuild after that last commit.

Rest LGTM! Thanks a lot 😄

**Review by DanielNoord — APPROVED** [review]

**Comment by domoddball** [comment]
everything was good without the rebuild?

**Comment by DanielNoord** [comment]
Well, the CI test to check if the build was okay passed.

But this is because we're not actually showing the release notes in the `Info` tab for some reason 😅

**Pull request merged**